modules/develop/pages/data-transforms/deploy.adoc

```diff
@@ -0,0 +1,133 @@
+
+[[build]]
+== Build the Wasm binary
+
+To build a Wasm binary:
+
+1. Ensure your project directory contains a `transform.yaml` file.
+2. Build the Wasm binary using the xref:reference:rpk/rpk-transform/rpk-transform-build.adoc[`rpk transform build`] command.
+
+[source,bash]
+----
+rpk transform build
+----
+
+You should now have a Wasm binary named `<transform-name>.wasm`, where `<transform-name>` is the name specified in your `transform.yaml` file. This binary is your data transform function, ready to be deployed to a Redpanda cluster or hosted on a network for others to use.
+
+[[deploy]]
+== Deploy the Wasm binary
+
+You can deploy your transform function using the xref:reference:rpk/rpk-transform/rpk-transform-deploy.adoc[`rpk transform deploy`] command.
+
+. Validate your setup against the pre-deployment checklist:
++
+[%interactive]
+- [ ] Do you meet the <<prerequisites>>?
+- [ ] Does your transform function access any environment variables? If so, make sure to set them in the `transform.yaml` file or in the command-line when you deploy the binary.
+- [ ] Do your configured input and output topics already exist? Input and output topics must exist in your Redpanda cluster before you deploy the Wasm binary.
+
+. Deploy the Wasm binary:
++
+[source,bash]
+----
+rpk transform deploy
+----
+
+When the transform function reaches Redpanda, it starts processing new records that are written to the input topic.
+
+[[reprocess]]
+=== Reprocess records
+
+In some cases, you may need to reprocess records from an input topic that already contains data. Processing existing records can be useful if, for example, the output topic was accidentally deleted, and you need to recreate the lost data by reprocessing those existing records.
+
+You can specify the exact point from which the transform function should start processing records in the input topic, starting from a specific offset or a specific timestamp. These options ensure that you can handle various scenarios where reprocessing is necessary to maintain data integrity and resilience.
+
+To deploy a transform function and start processing records from a specific offset, use the following flag:
+
+[source,bash]
+----
+rpk transform deploy --from-offset <offset>
+----
+
+Example:
+
+[source,bash]
+----
+rpk transform deploy --from-offset 12345
+----
+
+To deploy a transform function and start processing records from a specific timestamp, use the following flag:
+
+[source,bash]
+----
+rpk transform deploy --from-timestamp <unix-timestamp>
+----
+
+Example:
+
+[source,bash]
+----
+rpk transform deploy --from-timestamp 1617181723
+----
+
+=== Share Wasm binaries
+
+You can also deploy data transforms on a Redpanda cluster by providing an addressable path to the Wasm binary. This is useful for sharing transform functions across multiple clusters or teams within your organization.
+
+For example, if the Wasm binary is hosted at `https://my-site/my-transform.wasm`, use the following command to deploy it:
```

Comment:
The URL renders as a clickable link, we could consider changing the formatting so that it's just a formatted string.